scripts/import_assets/import_assets.py

```diff
@@ -0,0 +1,85 @@
+import csv
+import json
+import requests
+import argparse
+import os
+from requests.exceptions import HTTPError
+from urllib.parse import urljoin
+
+DEFAULT_API_URL = 'https://se-api.holmsecurity.com/v1/'
+
+
+def read_data(args):
+    # reads data from a csv_file
+
+    with open(args.csv_path, 'r') as csv_file:
+        csv_reader = csv.reader(csv_file)
+        for row in csv_reader:
+            ip = row[5]
+            asset_type = get_asset_type(ip)
+            name = row[0]
+            business_impact = row[1]
+            details = row[2]
+            hosts_personal_data = row[4]
+            tags = [t for t in row[3].split('|') if t]
+            dict_fields = {"name": name,
+                "type": asset_type,
+                "business_impact": business_impact,
+                "details": details,
+                "tags": tags,
+                "hosts_personal_data": str_to_bool(hosts_personal_data)}
+            if asset_type == 'network':
+                dict_fields.update({"ip_range": ip})
+            else:
+                dict_fields.update({"ip": ip})
+            try:
+                post_asset_request(args, dict_fields)
+                print(f"{name} with the ip {ip} was added successfully")
+            except HTTPError as err:
+                errors = json.loads(err.response.content)["errors"]
+                print(f"The asset could not be added because: {errors}")
+
+
+
+def str_to_bool(s):
+    if s in ['true', 'True']:
+         return True
+    elif s in ['false', 'False']:
+         return False
+    else:
+         raise ValueError
+
+
+def get_asset_type(ip_row):
+    # check the type of the asset
```

Comment:
Please add docstring explaining why we have two types of assets. It can be something like:

```
"""
Check the type of the asset by the IP/IP range. We support two types of assets:
- network  (eg. 192.168.0.1/24)
- host        (eg. 192.168.0.134)
"""
```

Reply:
Done!